src/rt-app.c

```diff
@@ -37,6 +37,14 @@ Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 #include "rt-app_utils.h"
 #include "rt-app_args.h"
 
+/*
+ * To prevent infinite loops in fork bombs, we will limit the number of
```

Comment:
What if if really want to test a fork bomb?

Maybe we can use this just as a default value for a json configuration option... at least we keep the possibilty to set a very high number if that's the intent...